zcash_primitives/CHANGELOG.md

```diff
@@ -37,6 +41,14 @@ and this library adheres to Rust's notion of
     `orchard::builder::InProgress<orchard::builder::Unproven, orchard::builder::Unauthorized>`
     instead of `zcash_primitives::transaction::components::orchard::Unauthorized`
 - `zcash_primitives::consensus::NetworkUpgrade` now implements `PartialEq`, `Eq`
+- `zcash_primitives::legacy::OpCode` now defines a complete set of transparent opcodes,
+  instead of just a minimal set.
```

Comment:
`OpCode` is a private type. The only thing that this change effects in the public API is the `Debug` impl of `Script`.

Reply:
Oh, huh, I didn't realize it was private. Okay.

Reply:
Amended.